Training2/thongnq-training2/src/App.js

```diff
@@ -0,0 +1,53 @@
+import React, {useState, useRef, useEffect} from "react";
+
+import Button from "./task3/Button";
+
+function App() {
+
+  const [userInput, setUserInput] = useState(0);
+
+  const [error, setError] = useState("");
+
+  const [stopTimer, setStopTimer] = useState(false);
+
+  useEffect(() => {
+    const timer = userInput > 0 && !stopTimer && setInterval(() => setUserInput(userInput - 1), 1000);
+    return () => clearInterval(timer);
```

Comment:
@Thong-Nguyen-Zigvy  your interval always clear whenever the userInput changes, it's not good, the interval should be created when
- the user starts the timer with valid userInput
- the timer is stopped and the user tries to click the resume 
should be cleared when
- the user clicks stop  
- the timers have userInput decrease to 0
Please refactor this